File: NirvanaSMS.Api.CSharp/Core/Entity/ApiUser.cs
using NirvanaSMS.Api.Core.Base;

namespace NirvanaSMS.Api.Core.Entity
{
    public class ApiUser : UserBase
    {
        public ApiUser()
        {
          
        }

        public ApiUser(string userName, string apiPassword)
        {
            UserName = userName;
            ApiPassword = apiPassword;
        }
    }
}

File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/entity/ApiUser.java
package org.nirvanasms.api.core.entity;

import org.nirvanasms.api.core.base.UserBase;

public class ApiUser extends UserBase {
	 public ApiUser()
     {
       
     }

     public ApiUser(String userName, String apiPassword)
     {
         super.setUserName(userName);
         super.setApiPassword(apiPassword);
     }
}
